### PR TITLE
SignIn and SignUp are working

### PR DESCRIPTION
**Pull request by KapJ1coH**

### CODE REVIEW

File: peer-assessment-app/app/api/auth/[...nextauth]/route.ts [hunk 1]
```diff
@@ -1,55 +1,94 @@
...
+                    } else {
+                        // Invalid user type
+                        return null;
+                    }
+
+                    const [rows]: any = await db.execute(query, params);
+
+                    if (rows.length === 0) {
+                        // No user found
+                        return null;
```
Comment (M-a-a-d-man) [inline peer-assessment-app/app/api/auth/[...nextauth]/route.ts:64]: return null? should you not return a next response instead?

Comment (KapJ1coH) [reply peer-assessment-app/app/api/auth/[...nextauth]/route.ts:64]: The null return is handled by 
if (!result || result.error) {
            console.error('Failed to sign in');
            return;
        } else {
            console.log('Signed in');
            if (data.get('userType') === 'student') {
                router.push('/student');
            } else if (data.get('userType') === 'instructor') {
                router.push('/instructor');
            }
        }

Comment (KapJ1coH) [reply peer-assessment-app/app/api/auth/[...nextauth]/route.ts:64]: or am I wrong

**Review by M-a-a-d-man — APPROVED** [review]
problems resolved.

**Pull request merged**